dbt_core_integration.py

```diff
@@ -110,6 +110,37 @@ def validate_sql(
         raise Exception(str(e))
 
 
+def project_healthcheck(manifest_path, catalog_path=None, config_path=None, config=None):
+    try:
+        from datapilot.config.config import load_config
+        from datapilot.core.platforms.dbt.constants import MODEL
+
+        # from datapilot.core.platforms.dbt.constants import PROJECT
+        from datapilot.core.platforms.dbt.executor import DBTInsightGenerator
+        import logging
+        import json
+
+        logging.basicConfig(level=logging.INFO)
+
+        if not config and config_path:
+            config = load_config(config_path)
+        insight_generator = DBTInsightGenerator(
+            manifest_path,
+            catalog_path=catalog_path,
+            config=config,
+        )
+        reports = insight_generator.run()
+
+        # package_insights = reports[PROJECT]
+        model_insights = {
+            k: [json.loads(item.json()) for item in v]
+            for k, v in reports[MODEL].items()
+        }
+        return {"model_insights": model_insights}
+    except Exception as e:
+        raise Exception(str(e))
+
+
```

Comment:
If we only support core it means we need to block this feature for dbt cloud.

Reply:
we are also doing this for dbt cloud. It should behave the same but haven't testes it yet.